### DISCUSSION TITLE
Can I prevent Visual (RStudio) from putting \begin{} latex commands in a tex code block?

### DISCUSSION THREAD

**Discussion by lucdw** [quarto-dev/quarto-cli#6803] [Q&A] [answered]
### Description

The file below doesn't has double dollar signs around the align block, because I have to render it to html **and** to pdf.

```` {qmd}
---
title: "TestAlign"
---

## Checkalign

\begin{align}
y_1 &= a x_1 + c x_2 \\
y_2 &= b x_1 + d x_2
\end{align}

````

When I look at the file in the 'Visual' mode, RStudio puts the align block in a tex code block, so it doesn't render any more ...

Is there a way to prevent RStudio from doing this ?

**Comment by mcanouil** [comment]
Please edit, the title to remove `<>` and please format your post properly to be legible.
Thank you!

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by lucdw** [reply]
Done

**Comment by cscheid** [comment, marked as answer]
This is a Pandoc issue, unfortunately: https://github.com/jgm/pandoc/issues/7968
